lib/matplotlib/pyplot.py

```diff
@@ -1274,6 +1274,86 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
     return fig, axs
 
 
+def subplot_mosaic(layout, *, subplot_kw=None, gridspec_kw=None,
+                   empty_sentinel='.', **fig_kw):
```

Comment:
not thrilled that all the docs need to be repeated here, is it so help? works

Reply:
It is also a slightly different API because we also create the figure.  Having the duplication is annoying, but I don't think there is a mechanical way to do the re-writing that is simpler in the long run that just having the duplicate text.